src/common/utils.js

```diff
@@ -173,8 +173,8 @@ const request = (data, headers) => {
  * @param {object} props Function properties.
  * @param {string[]} props.items Array of items to layout.
  * @param {number} props.gap Gap between items.
- * @param {number[]?=} props.sizes Array of sizes for each item.
- * @param {"column" | "row"?=} props.direction Direction to layout items.
+ * @param {number[]=} props.sizes Array of sizes for each item.
+ * @param {"column" | "row"=} props.direction Direction to layout items.
```

Comment:
```suggestion
 * @param {number[]=} props.sizes Array of sizes for each item.
```

Reply:
Maybe we can swap the order so that it is the same as in the function description?

Reply:
Good suggestion, i have updated the code, check it please.